### PR TITLE
ThreatConnect - Added reliability

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
related: demisto/etc#29347

## Description
Added reliability

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Deprecated. Use the ThreatConnect v2 integration instead.
 | accessId | Access ID | True |
 | secretKey | Secret Key | True |
 | defaultOrg | Default Organization | False |
+| Source Reliability | Reliability of the source providing the intelligence data. The default value is B - Usually reliable. | True |
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/README.md:15]:
```suggestion
| Source Reliability | Reliability of the source providing the intelligence data. The default value is: B - Usually reliable. | True |
```

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 2]
```diff
@@ -130,6 +130,7 @@ def create_context(indicators, include_dbot_score=False):
                     'Type': indicator_type,
                     'Vendor': 'ThreatConnect'
                 }
+            indicators_dbot_score[value]['Reliability'] = params.get('integrationReliability')
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:133]: it is not a good practice to call params.get() within a func, please initilalize at `GLOBAL VARS` and call as global.

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 3]
```diff
@@ -16,6 +16,13 @@
 '''GLOBAL VARS'''
 FRESHNESS = int(demisto.params().get('freshness', 0))
 MAX_CONTEXT = 100
+
+reliability = demisto.params().get('integrationReliability')
+
+if DBotScoreReliability.is_valid_type(reliability):
+    reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
+else:
+    raise Exception("Please provide a valid value for the Source Reliability parameter.")
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:25]: use return_error, please raise only when you catch it.

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 4]
```diff
@@ -16,6 +16,13 @@
 '''GLOBAL VARS'''
 FRESHNESS = int(demisto.params().get('freshness', 0))
 MAX_CONTEXT = 100
+
+reliability = demisto.params().get('integrationReliability')
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:20]:
```suggestion
RELIABILITY = demisto.params().get('integrationReliability', 'B - Usually reliable')
```

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 5]
```diff
@@ -16,6 +16,13 @@
...
+
+if DBotScoreReliability.is_valid_type(reliability):
+    reliability = DBotScoreReliability.get_dbot_score_reliability_from_str(reliability)
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:23]:
```suggestion
    RELIABILITY = DBotScoreReliability.get_dbot_score_reliability_from_str(RELIABILITY)
```

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 6]
```diff
@@ -16,6 +16,13 @@
 '''GLOBAL VARS'''
 FRESHNESS = int(demisto.params().get('freshness', 0))
 MAX_CONTEXT = 100
+
+reliability = demisto.params().get('integrationReliability')
+
+if DBotScoreReliability.is_valid_type(reliability):
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:22]:
```suggestion
if DBotScoreReliability.is_valid_type(RELIABILITY):
```

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 7]
```diff
@@ -130,6 +137,7 @@ def create_context(indicators, include_dbot_score=False):
                     'Type': indicator_type,
                     'Vendor': 'ThreatConnect'
                 }
+            indicators_dbot_score[value]['Reliability'] = reliability
```
Comment (bakatzir) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:140]:
```suggestion
            indicators_dbot_score[value]['Reliability'] = RELIABILITY
```

**Review by bakatzir — APPROVED** [review]

File: Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py [hunk 8]
```diff
@@ -16,6 +16,14 @@
 '''GLOBAL VARS'''
 FRESHNESS = int(demisto.params().get('freshness', 0))
 MAX_CONTEXT = 100
+
+RELIABILITY = demisto.params().get('integrationReliability', 'B - Usually reliable')
```
Comment (JasBeilin) [inline Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py:20]: lets see if we can put it in the main section.

**Review by bakatzir — APPROVED** [review]
@yaakovpraisler please make sure that the master build passes so I will be able to merge.

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**